plugins/cpu/fu-cpu-device.c

```diff
@@ -10,30 +10,27 @@
 
 struct _FuCpuDevice {
 	FuDevice		 parent_instance;
-	gboolean		 has_shstk;
-	gboolean		 has_ibt;
+	FuCpuDeviceFlag		 flags;
```

Comment:
I think the two commits got squashed together.

Reply:
Not sure how that happened, just rebased again.